src/utils/editUtils.ts

```diff
@@ -46,3 +47,11 @@ async function executeRangeFormat(editor: TextEditor, startPosition: Position, l
     editor.selection = new Selection(startPosition, endPosition);
     await commands.executeCommand("editor.action.formatSelection");
 }
+
+export function getIndentation(document: vscode.TextDocument, offset: number): string {
```

Comment:
In this PR, why do you extract it here? I don't see any other caller

Reply:
there is another caller in addDependencyHandler.ts. Because I need to use it in excludeDependencyHandler, so I extract the function from addDependencyHandler.ts to editUtils